src/resources/js/aggregator-fields.js

```diff
@@ -210,7 +213,8 @@ tribe_aggregator.fields = {
 			} )
 			.on( 'change', obj.selector.field_url_source, function( e ) {
 				var $field = $( this );
-				var value = $field.val();
+				var selectData = $field.select2( 'data' );
```

Comment:
The same on this one.

Reply:
Yeah we want to avoid moving too much to SelectWoo since we might come back to Select2 in the future if Select2 becomes accessible.